terraform/prod/nodepool.tf

```
@@ -134,3 +142,83 @@ resource "google_container_node_pool" "jenkins_workers_multiarch_c4a" {
     strategy        = "SURGE"
   }
 }
+
+# google_container_node_pool.jenkins_workers_multiarch_snowflake:
+resource "google_container_node_pool" "jenkins_workers_multiarch_snowflake" {
+  cluster            = "jenkins-test"
+  location           = "us-central1-c"
+  max_pods_per_node  = 110
+  name               = "jenkins-workers-multiarch-c4a"
+  initial_node_count = 0
+  autoscaling {
+    min_node_count = 0
+    max_node_count = 2
+  }
+  node_locations = [
+    "us-central1-a",
+  ]
+  project = "prompt-proto"
+
+  management {
+    auto_repair  = true
+    auto_upgrade = true
+  }
+
+  network_config {
+    create_pod_range     = false
+    enable_private_nodes = true
+    #pod_ipv4_cidr_block  = "10.224.0.0/14"
+    pod_range = "gke-jenkins-test-pods-ed1f94bc"
+  }
+
+  node_config {
+    disk_size_gb                = 200
+    disk_type                   = "hyperdisk-balanced"
+    enable_confidential_storage = false
+    image_type                  = "COS_CONTAINERD"
+    labels                      = {
+      workload = "snowflake"
+    }
+    taint                       = {
```

This should be a block.
```suggestion
    taint                       {
```